src/components/Analysis/Applications/CreateApplicationDialog.vue

```diff
@@ -69,22 +69,52 @@
           <template #label>
             CPU <span class="label-helper"> required </span>
           </template>
-          <el-input
+          <!-- <el-input
```

Comment:
same goes here - if you are going to leave commented out code, please put a comment explaining why it's there @SamanthaKraft - no requested changes this time, just a reminder.

Reply:
I will remove it thank you!